src/js/tabs/pid_tuning.js

```diff
@@ -759,6 +766,28 @@ TABS.pid_tuning.initialize = function (callback) {
         self.updatePIDColors();
     }
 
+    function sliderLevelView(sliderLevel) {
+        if (sliderLevel == 1) {
+            $('.tab-pid_tuning .PDRatioSlider').show();
+            $('.tab-pid_tuning .PDGainSlider').show();
+            $('.tab-pid_tuning .ResponseSlider').show();
```

Comment:
These lines are used in both cases, are they hidden otherwise?

Reply:
Oh oh - WIP - premature review ;-)

Reply:
No, please!  I'll take all the help I can get!!!  I need some and hence why I posted it as a draft so everyone can see I'm working on something and also to make it easier to hopefully get some help to stop sitting here and staring at the code for 3 or 4 hours at a shot.  

My biggest thing right now I think is the Configurator bug(?) about it not holding variables on saves.  I need to also figure out how to pass local multiple variables to the `slideLevelView` function.  I need it to operate on the `sliderLevelSelect` dropdown and also the D_min toggle; hence why I broke out to its own function.  I reckon i just need to list both variables in each call with a comma separator?  That's where I left off last night.

Thanks in advance for your patience with me!!  Dumb-dumb brain here when it comes to this stuff, but I'm trying.  I do know some stuff, but my C++ days were 20-yrs ago and I hated that class in college.  "I know enough to be dangerous".  I wish to know more to help out.

Reply:
@spatzengr are you trying with firmware master branch?

Reply:
Sliders in 1.44 do work, not with older firmware.

Reply:
@IllusionFpv , I'm on a build from 1/5/2020 for BF version.  Once @haslinghuis fixed gets merged (#2401), I will rebase the branch on Configurator Master and flash the latest BF Master for vetting.